fe/fe-core/src/main/java/com/starrocks/catalog/ExternalOlapTable.java

```diff
@@ -200,13 +206,15 @@ public void write(DataOutput out) throws IOException {
         Text.writeString(out, type.name());
         out.writeLong(id);
         Text.writeString(out, name);
+        out.writeLong(sourceDbId);
```

Comment:
Better to use json to persist, or you must increase the meta version if you add more persist member.

Reply:
ok